daemon/list.go

```diff
@@ -516,6 +517,21 @@ func (daemon *Daemon) filterVolumes(vols []volume.Volume, filter filters.Args) (
 				continue
 			}
 		}
+		if filter.Include("label") {
+			v, err := daemon.volumes.Get(vol.Name())
```

Comment:
@HackToday you're correct, it does not return the labels, it's what we're discussing here. From my point of view, `List` should return the labels itself.